components/Button.tsx

```diff
@@ -23,7 +23,7 @@ const Button: FC<ButtonProps> = ({ type, key, children }) => {
       className={`${colorVariants[type]} flex content-center items-center justify-center space-x-1.5 rounded-md px-1`}
     >
       {children}
-      <p className="flex">{type}</p>
+      {type !== 'Hug' ? <p className="flex">{type}</p> : null}
```

Comment:
Why we use a tag in the component that its name is button?
also what do you think about this:
`{type !== 'Hug' && <p className="flex">{type}</p>}`